chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -237,6 +176,29 @@
 						],
 						"type": "text/javascript"
 					}
+				},
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"var ReqBody= {\r",
+							"        userName:pm.environment.get(\"username\"),password:pm.environment.get(\"password\")\r",
```

Comment:
We don't have environment property as username and password. We have only usernameACB and passwordACB.
I am not aware of security of this controller but if it is supposed to use ACB credentials then these usernameACB and passwordACB should be used. If it is going to use other credentials, then need to add those keys to sample.postman_environment.json and also update bamboo.